.github/workflows/test.yml

```diff
@@ -7,12 +7,12 @@ jobs:
       # The CKAN version tag of the Solr and Postgres containers should match
       # the one of the container the tests run on.
       # You can switch this base image with a custom image tailored to your project
-      image: openknowledge/ckan-dev:2.9
+      image: openknowledge/ckan-dev:2.10
     services:
       solr:
         image: ckan/ckan-solr-dev:2.9
```

Comment:
Noticed the solr has not been updated to 2.10 as well, any reason why this image has not been updated as well?
Just checking if this was intentional or it just passed unoticed.
There are 2.10 versions available --> https://github.com/ckan/ckan-solr

Reply:
Also noticed that the GitHub Actions are failing due to the image `ckan/ckan-solr-dev:2.9`  no longer being available.
See here https://hub.docker.com/search?q=ckan%2Fckan-solr you will get a 404.